index.js

```diff
@@ -56,7 +57,7 @@ exports.bytes = encoder(2,
     var len = varint.decode(buffer, offset)
     offset += varint.decode.bytes
 
-    var val = buffer.slice(offset, offset + len)
+    var val = buffer.subarray(offset, offset + len)
```

Comment:
May want to mention (in the eventual squashed commit) that Node.js >= 16.15.0 is required to avoid a performance regression, because 16.15.0 made `Buffer#subarray` an alias for `Buffer#slice`, replacing `TypedArray#subarray` which is significantly slower.

Reply:
Thanks for mentioning, I wasn't aware of this. I've added a note to the ReadMe, just below the npm install instructions